### ISSUE TITLE
Received a 400 HTTP response while deleting comments

### ISSUE THREAD

**Issue by refekt** [Nick-Gottschlich/Social-Amnesia#4]
https://imgur.com/FQRPJwU

It happens every time I attempt it.

**Comment by Nick-Gottschlich** [comment]
We chatted a bit about this, my initial guess is this has to do with some kind of character than tkinter or PRAW doesn't like (emoji?) or something gets tangled up when going through large amount of comments. I don't have a tester account with more than a 100 items, but I'll see if I can replicate this just by scraping a reddit account with lots of items.

**Comment by Nick-Gottschlich** [comment]
```
BEFORE ITEM SNIPPETWiped by Social Amnesia
BEFORE CREATED_UTCWiped by Social Amnesia
AFTER CREATED_UTCWiped by Social Amnesia
score-5Wiped by Social Amnesia
gilded0Wiped by Social Amnesia
(<class 'prawcore.exceptions.BadRequest'>, BadRequest('received 400 HTTP response',), <traceback object at 0x10b38ec08>)
<class 'prawcore.exceptions.BadRequest'>
received 400 HTTP response
<traceback object at 0x10b38ec08>
```

It looks to me like the reddit API might be retrieving a deleted comment... I can probably fix this with a try/catch. Not the ideal solution but it will at least stop these crashes.

**Comment by Nick-Gottschlich** [comment]
Fixed in #5
